android/src/main/java/io/wifi/p2p/WiFiP2PManagerModule.java

```diff
@@ -34,6 +35,7 @@
     private ReactApplicationContext reactContext;
     private final IntentFilter intentFilter = new IntentFilter();
     private WiFiP2PDeviceMapper mapper = new WiFiP2PDeviceMapper();
+    private String groupPassword;
```

Comment:
May I ask why do you decide to keep this inforamtion in class?
I think we can declare this variable locally